From 039ca2d460d05a007139f664b6185d6e8a85e53d Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <l.majewski@samsung.com>
Date: Tue, 4 Mar 2014 12:29:04 +0100
Subject: [PATCH 1290/1302] cpufreq:LAB: Remove MODULE_* macros since it is not
 possible to compile LAB as a module

LAB cannot be compiled as a module, so MODULE_* macros are in fact a dead
code.

Change-Id: I10f8e5650631b00850631aea1295c1d88e53104a
Signed-off-by: Lukasz Majewski <l.majewski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/cpufreq/cpufreq_lab.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/cpufreq/cpufreq_lab.c b/drivers/cpufreq/cpufreq_lab.c
index 26360c3..1738a7a 100644
--- a/drivers/cpufreq/cpufreq_lab.c
+++ b/drivers/cpufreq/cpufreq_lab.c
@@ -450,12 +450,6 @@ static int __init cpufreq_gov_dbs_init(void)
 	return cpufreq_register_governor(&cpufreq_gov_lab);
 }
 
-MODULE_AUTHOR("Jonghwa Lee <jonghwa3.lee@samsung.com>");
-MODULE_AUTHOR("Lukasz Majewski <l.majewski@samsung.com>");
-MODULE_DESCRIPTION("'cpufreq_lab' - A dynamic cpufreq governor for "
-		"Legacy Application Boosting");
-MODULE_LICENSE("GPL");
-
 #ifdef CONFIG_CPU_FREQ_DEFAULT_GOV_LAB
 fs_initcall(cpufreq_gov_dbs_init);
 #else
-- 
1.8.3.2

